src/psyclone/psyGen.py

```diff
@@ -6777,6 +6883,107 @@ def _ignore_handler(self, *_):
         '''
         return None
 
+    def _create_loop(self, parent, variable_name):
+        '''
+        Create a Loop instance. This is done outside _do_construct_handler
+        because some APIs may want to instantiate a specialized Loop.
+
+        :param parent: The parent of the node.
```

Comment:
Lower-case start to `:param:` and `:return:` descriptions please.

Reply:
fixed